src/coreclr/debug/daccess/daccess.cpp

```diff
@@ -7045,6 +7045,12 @@ CLRDataCreateInstance(REFIID iid,
                 // Release the AddRef from the QI.
                 pClrDataAccess->Release();
             }
+
+            if (cdacInterface == nullptr)
```

Comment:
@max-charlamb Just curious, the comment on this method mentions it’s a legacy entrypoint. What’s the modern or recommended alternative for cDAC? And what would it take to switch over to it?